Microsoft.Toolkit.Uwp.SampleApp/SamplePages/IconExtensions/IconExtensionsPage.xaml

```diff
@@ -0,0 +1,9 @@
+﻿<Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.IconExtensionsPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+    <Grid>
```

Comment:
You should add the markup extensions use here so that they don't get optimized out by the .NET Native compiler in release mode for the sample app, see the readme.

Reply:
Whoops, my bad! Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3110/commits/bf6a306d0e79b267117f4c182e4c50ce59c918bb, thanks! 😄

Reply:
No problem, it's an obscure requirement. 😋